internal/factsengine/gatherers/sapservices.go

```diff
@@ -0,0 +1,187 @@
+package gatherers
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"regexp"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/afero"
+	"github.com/trento-project/agent/pkg/factsengine/entities"
+)
+
+type SapServicesStartupKind string
+
+const (
+	SapServicesSystemdStartup  SapServicesStartupKind = "systemctl"
+	SapServicesSapstartStartup SapServicesStartupKind = "sapstartsrv"
+	sapServicesDefaultPath                            = "/usr/sap/sapservices"
+	SapServicesGathererName                           = "sapservices"
+)
+
+// nolint:gochecknoglobals
+var (
+	SapServicesParsingError = entities.FactGatheringError{
+		Type:    "sap-services-parsing-error",
+		Message: "error parsing the sap services file",
```

Comment:
I would join the `sap services` to `sapservices`. In the `Type` and in the `Message`.
The same for the next errors

Reply:
done